core/src/avm2/globals/flash/system/system.rs

```diff
@@ -11,8 +11,17 @@ pub fn set_clipboard<'gc>(
     _this: Option<Object<'gc>>,
     args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
-    //TODO: in FP9+ this function only works when called from a button handler in the Plugin due to
-    // sandbox restrictions
+    // TODO: In the plugin, this function only works when called as a result of a user interaction
+    // due to sandbox restrictions.
+    #[cfg(target_family = "wasm")]
+    if false {
+        return Err(Error::AvmError(crate::avm2::error::io_error(
+            activation,
+            "Error #2176",
```

Comment:
nit: Provide the error message as well ("Error #2176: Certain actions, such as those that display a pop-up window, may only be invoked upon user interaction, for example by a mouse click or button press.")


<br class="Apple-interchange-newline">")

Reply:
Thanks, I added the full message. Still worth to mention that the FP plugin only displayed `Error #2176` but this behavior likely occurs for any other error.